src/__test__/home.test.js

```diff
@@ -0,0 +1,19 @@
+import '@testing-library/jest-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import Home from '../components/home';
+import store from '../redux/configureStore';
+
+describe('Home component testing', () => {
+  test('Does the Home component match snapshot', () => {
+    const home = render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <Home />
+        </Provider>
+      </BrowserRouter>,
+    );
+    expect(home).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Please add more unit tests that can test the pure functions from your Redux code, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests%20for%20pure%20functions%20in%20your%20Redux%20code.) requirement.
- [ ] Please make sure to set up integration tests for your project that test how the data is sent, received, and handled in your app, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests,the%20Redux%20Store.) requirement.